Subject: [PATCH] SystemUI: Fix SystemUI Crash

SystemUI may crash sometimes in monkey test due to exception.

Catch the exception to avoid the crash.

diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/policy/UserInfoControllerImpl.java b/packages/SystemUI/src/com/android/systemui/statusbar/policy/UserInfoControllerImpl.java
old mode 100644
new mode 100755
index b1e4b03..4f41797
--- a/packages/SystemUI/src/com/android/systemui/statusbar/policy/UserInfoControllerImpl.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/policy/UserInfoControllerImpl.java
@@ -111,7 +111,12 @@
             mUserInfoTask.cancel(false);
             mUserInfoTask = null;
         }
-        queryForUserInformation();
+
+        try {
+            queryForUserInformation();
+        } catch (Exception e) {
+            Log.e(TAG, "Couldn't query user info", e);
+        }
     }
 
     private void queryForUserInformation() {
